lsp-common.el

```diff
@@ -28,6 +28,11 @@
   :group 'lsp-mode
   :type 'boolean)
 
+(defvar lsp--uri-file-prefix (pcase system-type
```

Comment:
Wouldn't using `eval-when-compile` be a better idea here?

Reply:
Compiled elisp files should be portable across operating systems and a `eval-when-compile` in this case would violate this condition.

Reply:
ah, okay